pkg/server/nfs/nfs_server.go

```diff
@@ -102,10 +115,20 @@ func NewServer(logger logrus.FieldLogger, configPath, exportPath, volume string)
 	}, nil
 }
 
+func SetConfigParams() {
```

Comment:
This is good, but it's not granular but rather a global setting instead.

Because ganesha has several components, if all use `FULL_DEBUG`, it will possibly cause unexpected log flooding we don't know yet. Suggest not doing it right now, but as I said, just focus on the log redirection.

Reply:
I will focus on #38 and waiting more discussion to make the solution precisely.